### DISCUSSION TITLE
需不需要规范一下内容？

### DISCUSSION THREAD

**Discussion by Typhonling** [xlucn/oh-my-foss-android#15] [想法]
一是fdroid链接，使用 https://f-droid.org/en/* 还是去掉en使用 https://f-droid.org/* ?
二是软件名称，有少部分只有中文名称。是否统一采用主英文/副中文的形式。

**Comment by xlucn** [comment]
感谢提供建议

> 一是fdroid链接，使用 https://f-droid.org/en/* 还是去掉en使用 https://f-droid.org/* ?

不清楚有何区别，比如会匹配系统/浏览器语言？如果有区别的话可以改过来，之前有en仅仅是因为我都是从浏览器地址栏复制的

> 二是软件名称，有少部分只有中文名称。是否统一采用主英文/副中文的形式。

我自己加的都是使用英文为主，如果开发者自己起了中文名称，我也会加上。统一都加fdroid上中文名的话，操作就多了一些，但也可以考虑。但问题在于大部分都不是开发者自己起的名字，而是他人贡献的翻译，一是可能不全，二是更容易变化不便更新

**Comment by Typhonling** [comment]
> 不清楚有何区别，比如会匹配系统/浏览器语言？如果有区别的话可以改过来，之前有en仅仅是因为我都是从浏览器地址栏复制的

区别是en显示英语，没有添加语言标识会根据用户环境自动匹配语言。

> 但问题在于大部分都不是开发者自己起的名字，而是他人贡献的翻译，一是可能不全，二是更容易变化不便更新

我就是说有些软件只标识了中文名，不便识别，可以补上英文名称。另外补充些中文名称的话也许能更好地在搜索引擎中引流？

那有空我会提交PR。中文名我主要以Google Play商店中的中文命名做参考吧，有些不知所谓`RadioDroid/睿卓`或者太泛用化`Loop Habit Tracker/习惯`的命名就算了。

**Comment by Typhonling** [comment]
另外RadioDroid这个软件，我当时直接放在另外两款收音机下方了，但自带内容不符合`内容获取`这个分类的描述。你看看该移动到哪里吧。

**Reply by xlucn** [reply]
pr看到了，感谢，我等下看看。

这个分类的话，先这样放着吧。我刚开始的分类也没有精心设计过，以后可以考虑改分类的名称或者外延 。
